dev-util/sysdig/sysdig-0.1.104.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit linux-mod bash-completion-r1 cmake-utils
+
+DESCRIPTION="A system exploration and troubleshooting tool"
+HOMEPAGE="http://www.sysdig.org/"
+SRC_URI="https://github.com/draios/sysdig/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+modules"
+
+RDEPEND="dev-lang/luajit:2=
+	>=dev-libs/jsoncpp-0.6_pre:0=
+	sys-libs/zlib:0=
+	>=sys-libs/ncurses-6.0-r1:0="
```

Comment:
On Mon, Sep 21, 2015 at 2:56 AM, Michał Górny <notifications@github.com>
wrote:

> Just confirmed ncurses-5:0 builds fine. So, unless you've got a compelling
> reason to require 6, I'd suggest you leave it at slot.


As in sys-libs/ncurses:0, correct?

On Mon, Sep 21, 2015 at 2:53 AM, Michał Górny <notifications@github.com>
 wrote:

> you ... didn't update the Manifest.


You're right... did SOP change for this?  I was "sure" they were in
.gitignore, but they sure ain't now.

I'll touch it up; I think github will silently update my PR so I'll ping
you OOB when it's all in order.